app/src/main/java/uk/gov/register/FieldsConfiguration.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.register;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.google.common.collect.Lists;
+import io.dropwizard.jackson.Jackson;
+
+import java.io.*;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.logging.Logger;
```

Comment:
[dropwizard uses logback for logging](http://www.dropwizard.io/0.9.1/docs/manual/core.html#logging) so why have you used JUL here?